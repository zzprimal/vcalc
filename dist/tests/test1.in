print(5607);
int AuGAA45YJwShECfgb = 9288 / 20797 - 31535;
print(31949);

//CHECK_FILE:./test1.out
