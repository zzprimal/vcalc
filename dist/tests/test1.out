5607
31949
